red-hat-data-services/odh-dashboard
# [UPSTREAM] BYON feature

- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-3059

TEST STEPS:

Login to the Dashboard as a user in the RHODS admin group dedicated-admins
Verify Settings -> Notebook Images is enabled
Click on Notebook Images
Click Import Image and specify any valid notebook image repo url, notebook name with OPTIONAL description, software and packages
Click Import
Verify that the Imagestream and tag was created
Login to JH and spawn a notebook with that custom image
